slsim/LsstSciencePipeline/util_lsst.py

```diff
@@ -210,3 +214,373 @@ def opsim_variable_lens_injection(
     exposure_data_new.add_columns([lens_col, final_image_col])
 
     return exposure_data_new
+
+
+def optimized_transient_event_time_mjd(
+    mjd_times, lightcurve_range=(-50, 100), min_points=100, optimized_cadence=True
+):
+    """Finds a transient start time such that at least min_points fall within
```

Comment:
I don't really understand what min_points means and what this function exactly does from the documentation